js_modules/dagster-ui/packages/ui-core/src/automation/__stories__/AutomationTargetList.stories.tsx

```diff
@@ -17,10 +25,9 @@ export default {
   component: AutomationTargetList,
 } as Meta;
 
-export const AutomationTargetListWithError = () => {
+export const SingleJob = () => {
   return (
-    <>
-      <CustomAlertProvider />
+    <RecoilRoot>
```

Comment:
😰 I want to migrate from Recoil to Jotai since its actively maintained and supports React 19 and doesn't require a root provider